velox/expression/tests/SignatureBinderTest.cpp

```diff
@@ -57,7 +57,7 @@ TEST(SignatureBinderTest, decimals) {
         signature->argumentTypes()[1].toString(),
         "DECIMAL(b_precision, b_scale)");
     testSignatureBinder(
-        signature, {DECIMAL(11, 5), DECIMAL(10, 6)}, DECIMAL(12, 6));
+        signature, {DECIMAL(11, 5), DECIMAL(10, 6)}, DECIMAL(13, 6));
```

Comment:
Why is this change?

Reply:
This PR caught and fixed this bug as the type check changed from `kindEquals -> equivalent`.
I also manually verified that 13 is the correct answer.

Reply:
Nice. Would you mention this in the PR description?

Reply:
Thanks for the review comments, Masha. I addressed them.